src/ripple/json/impl/Writer.cpp

```diff
@@ -244,20 +243,20 @@ void Writer::output (std::string const& s)
 void Writer::output (Json::Value const& value)
 {
     impl_->markStarted();
-    writeJson (value, impl_->getOutput());
+    outputJson (value, impl_->getOutput());
 }
 
 template <>
 void Writer::output (float f)
 {
```

Comment:
@rec I'm concerned about these template specializations  in the cpp when there is already a definition in the header file.